### PR TITLE
Reviewed the Assetic cookbook articles

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | - |

### CODE REVIEW

File: cookbook/assetic/uglifyjs.rst [hunk 1]
```diff
@@ -16,41 +16,47 @@ talked about briefly.
...
+add the ``node_modules`` folder to version control. Alternatively, you can create
+an npm `package.json`_ file and specify your dependencies there.
 
-    $ uglifyjs --help
+Now you can execute the ``uglifyjs`` command that lives in the ``node_modules``
+directory:
+
+.. code-block:: bash
 
     $ ./app/Resources/node_modules/.bin/uglifyjs --help
```
Comment (WouterJ) [inline cookbook/assetic/uglifyjs.rst]: for windows compatbility, the path should be in double quotes

Comment (javiereguiluz) [inline cookbook/assetic/uglifyjs.rst]: You mean like this?

```
$ "./app/Resources/node_modules/.bin/uglifyjs" --help
```

Comment (WouterJ) [inline cookbook/assetic/uglifyjs.rst]: yes

2015-03-19 11:41 GMT+01:00 Javier Eguiluz <notifications@github.com>:

Comment (javiereguiluz) [inline cookbook/assetic/uglifyjs.rst]: Done. Thanks.

**Comment by WouterJ** [comment]
:+1:

File: cookbook/assetic/asset_management.rst [hunk 2]
```diff
@@ -204,7 +204,7 @@ Combining Assets
 ~~~~~~~~~~~~~~~~
 
 One feature of Assetic is that it will combine many files into one. This helps
-to reduce the number of HTTP requests, which is great for front end performance.
+to reduce the number of HTTP requests, which is great for frontend performance.
```
Comment (xabbuh) [inline cookbook/assetic/asset_management.rst]: Not sure about this change. While I agree that "frontend" looks most beautiful, it seems that "front-end" actually would be more correct (see also https://github.com/symfony/symfony-docs/pull/4671#discussion_r26449893).

File: cookbook/assetic/uglifyjs.rst [hunk 3]
```diff
@@ -231,8 +232,14 @@ the node package is installed:
 
 .. code-block:: bash
 
+    # global installation
     $ npm install -g uglifycss
 
+    # local installation
+    $ cd /path/to/your/symfony/project
+    $ mkdir app/Resources/node_modules
```
Comment (xabbuh) [inline cookbook/assetic/uglifyjs.rst]: Is this step needed? Doesn't `npm` create the directory itself if it does not exist?

File: cookbook/assetic/yuicompressor.rst [hunk 4]
```diff
@@ -170,4 +166,3 @@ apply this filter when debug mode is off.
 .. _`YUI Compressor`: http://developer.yahoo.com/yui/compressor/
 .. _`Download the JAR`: https://github.com/yui/yuicompressor/releases
 .. _`no longer maintained by Yahoo`: http://www.yuiblog.com/blog/2013/01/24/yui-compressor-has-a-new-owner/
```
Comment (xabbuh) [inline cookbook/assetic/yuicompressor.rst]: Should we change this to the other URL (that was removed)?

File: cookbook/assetic/asset_management.rst (hunk 2 above)
Comment (weaverryan) [inline cookbook/assetic/asset_management.rst]: both Bower and Bootstrap (on their homepages) use `front-end`. That looks a bit odd to me, but I guess we should follow their lead.

**Comment by weaverryan** [comment]
I love having a fresh set of eyes on these - thanks! @xabbuh I *did* remove the `mkdir` at sha: 20dcc3c63f229143194b0801741411b3cc5d6dc9. You're right - these are not needed.

Cheers!

File: cookbook/assetic/uglifyjs.rst (hunk 3 above)
Comment (WouterJ) [inline cookbook/assetic/uglifyjs.rst]: it does

**Pull request merged**